src/main/java/com/autotune/utils/KubeEventLogger.java

```diff
@@ -31,27 +33,29 @@
  * for kubernetes objects.
  */
 public class KubeEventLogger implements EventLogger {
-    private final KubernetesClient kubeClient;
     private final Clock clock;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(KubeEventLogger.class);
 
-    public KubeEventLogger(KubernetesClient kubeClient, Clock clock) {
-        this.kubeClient = kubeClient;
+    public KubeEventLogger( Clock clock) {
         this.clock = clock;
     }
 
     @Override
     public void log(String reason, String message, Type type, String objectName, String namespace, ObjectReference objectReference, String kind) {
+        KubernetesServices kubernetesServices = null;
         String componentName = AnalyzerConstants.AUTOTUNE;
-        String eventName = componentName + "." + ((reason + message + type + objectName).hashCode() & 0x7FFFFFFF);
-        Event existing = kubeClient.events().inNamespace(namespace).withName(eventName).get();
-        String timestamp = Instant.now(clock).toString();
         try {
+            kubernetesServices = new KubernetesServicesImpl();
+            String eventName = componentName + "." + ((reason + message + type + objectName).hashCode() & 0x7FFFFFFF);
+            //Event existing = kubeClient.events().inNamespace(namespace).withName(eventName).get();
+            Event existing = kubernetesServices.getEvent(namespace,eventName);
+            String timestamp = Instant.now(clock).toString();
             if (existing != null && existing.getType().equals(type.name()) && existing.getReason().equals(reason) && existing.getInvolvedObject().getName().equals(objectName) && existing.getInvolvedObject().getKind().equals(kind)) {
                 existing.setCount(existing.getCount() + 1);
                 existing.setLastTimestamp(timestamp);
-                kubeClient.events().inNamespace(namespace).withName(eventName).replace(existing);
+                kubernetesServices.replaceEvent(namespace,eventName,existing);
+                //kubeClient.events().inNamespace(namespace).withName(eventName).replace(existing);
```

Comment:
> kubernetesServices.replaceEvent(namespace,eventName,existing);
space after commas
//kubeClient.events().inNamespace(namespace).withName(eventName).replace(existing);
Remove this line

Reply:
Done